lms/djangoapps/teams/views.py

```diff
@@ -403,6 +407,28 @@ def post(self, request):
             return Response(CourseTeamSerializer(team).data)
 
 
+class IsEnrolledOrIsStaff(permissions.BasePermission):
```

Comment:
Note to reviewers: just a small amount of cleanup following @efischer19's PR, #9289.